client/my-sites/stats/activity-log/index.jsx

```diff
@@ -450,6 +451,7 @@ class ActivityLog extends Component {
 		} = this.props;
 
 		const disableRestore =
+			! enableRewind ||
 			includes( [ 'queued', 'running' ], get( this.props, [ 'restoreProgress', 'status' ] ) ) ||
 			'active' !== rewindState.state;
```

Comment:
It seems like the line
```
includes( [ 'queued', 'running' ], get( this.props, [ 'restoreProgress', 'status' ] ) )
```
is not working anymore, probably due to some refactoring of Rewind data fetching?

The prop `enableRewind` does something similar than this line, although it negates some sides. If we reconsider when `disableRestore` should be true, we have:
```
rewind is inactive || rewind is queued || rewind is running
```
so we can write it as:
```
const disableRestore = 'active' !== rewindState.state || includes( [ 'queued', 'running' ], restoreStatus );
```
and in line 624, instead of adding `enableRewind`, we could add `restoreStatus` (and destructure it here in `activityLog()`):
```
restoreStatus: get( rewindState, 'rewind.status', '' ),
```